templates/solutions/ubuntu-os/index.html

```diff
@@ -0,0 +1,653 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1X3osSqcm3rWLqYUJr5G4fXEEHCby-j5FEJwClLNx5S4/edit
+{% endblock meta_copydoc %}
+
+{% block title %}The world's favourite enterprise Linux platform{% endblock %}
+
+{% block meta_description %}
+  Ubuntu is the world's most popular enterprise Linux distribution. Unify your tech stack with the enterprise OS that is favoured for servers, IoT and applications.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Innovate with confidence.</h1>
+        <h2>Choose the world's favourite enterprise Linux.</h2>
+        <p class="p-section--shallow">
```

Comment:
```suggestion
      <div class="p-section--shallow">
        <p>
```
recently had a conversation about this with the Vanilla folk and the conclusion was to not to use positional class components ('.p-section') with content elements (`<p>`). I plan to add this to Vanilla documentation this pulse, but in the mean time, you can reference [the conversation on MM](https://chat.canonical.com/canonical/pl/yjjhnfdumtf5zm9ften6pghxeo)

Reply:
Sorry, not sure what happened here, usually pretty mindful about not doing this